translator/translate/otel/pipeline/containerinsights/translator.go

```diff
@@ -52,7 +53,7 @@ func (t *translator) Translate(conf *confmap.Conf) (*common.ComponentTranslators
 	if enhancedContainerInsightsEnabled {
 		return &common.ComponentTranslators{
 			Receivers:  common.NewTranslatorMap(awscontainerinsight.NewTranslator()),
-			Processors: common.NewTranslatorMap(metricstransformprocessor.NewTranslatorWithName(pipelineName), batchprocessor.NewTranslatorWithNameAndSection(pipelineName, common.LogsKey)), // EKS & ECS CI sit under metrics_collected in "logs"
+			Processors: common.NewTranslatorMap(metricstransformprocessor.NewTranslatorWithName(pipelineName), batchprocessor.NewTranslatorWithNameAndSection(pipelineName, common.LogsKey), gpu.NewTranslatorWithName(pipelineName)), // EKS & ECS CI sit under metrics_collected in "logs"
```

Comment:
Can you only add the processor if `gpu_metrics` is enabled?

Reply:
yup, updated.